backend/pick-git/src/test/java/com/woowacourse/pickgit/post/domain/comment/CommentTest.java

```diff
@@ -0,0 +1,45 @@
+package com.woowacourse.pickgit.post.domain.comment;
+
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+
+class CommentTest {
+
+    @DisplayName("100자 이하의 댓글을 생성할 수 있다.")
+    @Test
+    void newComment_Under100Length_Success() {
+        StringBuilder content = new StringBuilder();
+        for (int i = 0; i < 100; i++) {
+            content.append("a");
+        }
```

Comment:
```suggestion
        content.append("a".repeat(100));
```

이렇게 수정할 수 있을 거 같아요 !